src/Templates/OrchardCore.Cms.Templates/content/OrchardCore.Templates.Module/OrchardCore.Templates.Module.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Razor">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="OrchardCore.Module.Targets" Version="1.0.0-beta2-6657" />
```

Comment:
The version need to come from the `$(BuilderVersion)` that is set on the CI (it's also set to 1.0 locally)